civis/io/_tables.py

```diff
@@ -209,8 +217,24 @@ def read_civis_sql(sql, database, use_pandas=False, job_name=None,
     if archive:
         warnings.warn("`archive` is deprecated and will be removed in v2.0.0. "
                       "Use `hidden` instead.", FutureWarning)
+
+    if isinstance(database, str):
+        database = client.get_database_id(database)
```

Comment:
`client.get_database_id` should take care of handling whether an int or str is passed [1].  Is there a specific reason to check `isinstance` here?

https://github.com/civisanalytics/civis-python/blob/master/civis/civis.py#L45